### DISCUSSION TITLE
Update 20241207 unable to switch between output devices

### DISCUSSION THREAD

**Discussion by Jcauthron** [rern/rAudio#1835] [Fixed Issues]
I have two USB DACs connected and like to be able to switch between them. After updating to 20241206, I found that upon trying to switch output devices, it looks like it’s changing, and then goes back to the original one. They both work fine if they’re the only one plugged in, but with both plugged in, one seems to take priority. DAC in question are Schitt Modi Multibit and a Modi 3E. Rpi 3B+ Upgrade to 20241207 still same issue.  Thank you!

**Comment by rern** [comment]
Please try:
```sh
curl -Lo /srv/http/bash/settings/player-devices.sh https://github.com/rern/rAudio/raw/refs/heads/UPDATE/srv/http/bash/settings/player-devices.sh
```
